ts/Core/Templating.ts

```diff
@@ -187,7 +191,7 @@ function format(str = '', ctx: any, chart?: Chart): string {
         floatRegex = /f$/,
         decRegex = /\.(\d)/,
         lang = chart?.options.lang || defaultOptions.lang,
-        time = chart && chart.time || defaultTime,
+        timeInstance = chart && chart.time || time || defaultTime,
```

Comment:
`chart?.time || time || defaultTime` would be shorter.

You can also use the `.pick()` utils.